subjects/printandmiss/README.md

```diff
@@ -0,0 +1,49 @@
+## printandmiss
+
+### Instructions
+
+Write a function called `PrintAndMiss()` that takes a `string` and an `int` as an argument. The function should move through the `string` in sets determined by the `int`, printing the first set, omitting the second, printing the third, and so on, in a 'print' and 'miss' fashion until the end of the `string` is reached. Return a `string` containing the printed characters.
+
+- If the `string` is empty return `Invalid Output` followed by newline `\n`.
+- If the `int` is `0` or it's negative return the `string` followed by a newline `\n`.
```

Comment:
When it's a negative `int` it returns "Invalid Output" instead

Reply:
It was an instructional error, and I fixed it so the code is matching the instruction. Here's the correct instruction: "If the string is empty or the int is negative return Invalid Output followed by newline \n."

Reply:
fixed